src/main/resources/liquibase/tenant/scripts/2020-11-19--18-00-create-request-metadata-table.xml

```diff
@@ -0,0 +1,22 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+  <changeSet id="2020-11-19--18-00-create-request-metadata-table.xml" author="Illia Borysenko">
+    <createTable tableName="request_metadata_lb">
+      <column name="request_id" type="uuid">
+        <constraints primaryKey="true" nullable="false"/>
```

Comment:
should we also have a reference to the instance table with a foreign key?

Reply:
Yes, this makes sense, I added a foreign key constraint and change the logic of deleting instances respectively. Thank you.